### PR TITLE
czkawka: 3.2.0 -> 3.3.0

### PR DESCRIPTION
**Pull request by 0x4A6F**
###### Motivation for this change
- Upgrade to [3.3.0](https://github.com/qarmin/czkawka/releases/tag/3.3.0) incorporating these [changes](https://github.com/qarmin/czkawka/compare/3.2.0...3.3.0).
- Adding myself as maintainer
- Add installCheck

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by kira-bruneau — CHANGES_REQUESTED** [review]

**Comment by kira-bruneau** [comment]
Result of `nixpkgs-review pr 146757` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>czkawka</li>
  </ul>
</details>

**Comment by kira-bruneau** [comment]
Result of `nixpkgs-review pr 146757` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>czkawka</li>
  </ul>
</details>

**Review by kira-bruneau — APPROVED** [review]

File: pkgs/tools/misc/czkawka/default.nix [hunk 1]
```diff
@@ -36,10 +36,20 @@ rustPlatform.buildRustPackage rec {
...
+  doInstallCheck = true;
+  installCheckPhase = ''
+    if [[ "$("$out/bin/${pname}_cli" --version)" == "${pname} ${version}" ]]; then
+      echo '${pname} smoke check passed'
+    else
+      echo '${pname} smoke check failed'
+      return 1
+    fi
+  '';
+
```
[the remark this answers is not in the document]
Comment (0x4A6F) [reply pkgs/tools/misc/czkawka/default.nix]: Yeah, @jonringer infected me initially with this idea in [Nixpkgs - Adding a package to unstable/master branch](https://www.youtube.com/watch?v=fvj8H5yUKu8&t=1020s).

[the remark this answers is not in the document]
Comment (Artturin) [reply pkgs/tools/misc/czkawka/default.nix]: we have testVersion https://github.com/NixOS/nixpkgs/blob/eea701553efe6bf30f9e24262036df99bba5f1f8/pkgs/build-support/trivial-builders.nix#L724

[the remark this answers is not in the document]
Comment (0x4A6F) [reply pkgs/tools/misc/czkawka/default.nix]: Nice, testVersion was introduced in #121896.

**Pull request merged**